website/pages/docs/install.mdx

```diff
@@ -91,6 +99,7 @@ First things first, you'll need to install Million.js. You can do this with your
 
 Then, add the compiler to your build tool of choice:
```

Comment:
What is meant by `compiler`?

Reply:
The compiler is our compiler that does the optimizations; let me just change it to "million compiler"